resources/asciidoctor/spec/elastic_include_tagged_spec.rb

```diff
@@ -91,27 +83,26 @@ def expected_include(include_body)
     end
   end
   context "when including a file that doesn't exist" do
-    include_context 'convert'
-    let(:file) { 'resources/elastic_include_tagged/DoesNotExist.java' }
-    let(:input) do
-      "include::elastic-include-tagged:#{file}[doesn't-matter]"
+    let(:include_file) { 'resources/elastic_include_tagged/DoesNotExist.java' }
+    let(:tag)          { "doesn't-matter"                                     }
+    let(:expected_log) do
+      absolute_path = "#{__dir__}/#{include_file}"
+      "ERROR: <stdin>: line 5: include file not found: #{absolute_path}"
     end
     it 'the conversion contains a warning about unresolved directives' do
       expect(converted).to include(
-        "Unresolved directive in &lt;stdin&gt; - include::#{file}"
+        "Unresolved directive in &lt;stdin&gt; - include::#{include_file}"
       )
     end
     it 'logs a warning about the missing file' do
-      expect(logs).to eq(
-        "ERROR: <stdin>: line 2: include file not found: #{__dir__}/#{file}"
-      )
+      expect(logs).to include(expected_log)
     end
   end
   context "when including a tag that doesn't have a start tag" do
-    include_context 'convert'
-    let(:input) { include_input 'missing_start' }
+    let(:tag)              { 'missing_start' }
+    let(:expected_include) { ''              }
```

Comment:
also strange spacing

Reply:
I got this from [here](https://rubocop-rspec.readthedocs.io/en/latest/cops_rspec/#rspecalignrightletbrace) but I'll drop it.

Reply:
Oh, I thought maybe it was a typo. It looked strange but it could just be me!